README.md

```diff
@@ -3,11 +3,38 @@ A [Kanister](https://github.com/kanisterio/kanister) backup management system.
 
 Taweret sets retention periods for Kanister backups and deletes them once they expire by interacting with Kanister CRDs.
 
-This project is in a very early development phase. Please check the issues tracker for planned features, or to submit any feature requests.
+This project is in an early development phase. Please check the issues tracker for planned features, or to submit any feature requests.
 
 ## How to
 
-Taweret should be deployed to a Kubernetes cluster which already runs Kanister. A Taweret Helm chart does not yet exist.
+Taweret should be deployed to a Kubernetes cluster which already runs Kanister. 
+
+### Taweret
+
+Taweret can be installed through its Helm chart:
+
+    helm repo add renku https://swissdatasciencecenter.github.io/helm-charts
+    helm install taweret renku/taweret
+
+Backup configurations can be defined in the Helm values file. The default backup configuration is:
+
+    backupConfigs:
+      daily-postgres:
+        name: daily-postgres
+        kanisterNamespace: kanister
+        blueprintName: postgres-bp
+        profileName: default-profile
+        retention:
+          backups: 7
+          minutes: 0
+          hours: 0
+          days: 7
+          months: 0
+          years: 0
+
+The Taweret version which is installed can be set by specifying the image tag used by the Helm chart. To see the available image tags, please check the tags in the GitHub repo.
+
+Please be aware that the default image tag set in the Helm chart may not always be the most up to date Taweret image.
 
 ### Backup CronJob
```

Comment:
On some of the clusters, we saw that there was a `Completed` task which hung around after the backup had been completed - has this been addressed? Would it make sense to make a comment on this?

Reply:
Currently, the CronJobs which cause that left-over pod are not in Taweret's scope, but there is an issue open for possibly bringing that into Taweret's scope (#7)  Kanister ActionSets can be created in many different ways, though, so this is only really a problem if you use CronJobs to create the ActionSets.